src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2454,9 +2472,23 @@ because xbuild doesn't support framework reference assemblies.
 
   <Delete Files="$(_UploadFlagFile)" Condition="Exists ('$(_UploadFlagFile)')" />
 </Target>
-  
 
-<Target Name="_ResolveAndroidSigningKey" DependsOnTargets="$(_OnResolveMonoAndroidSdks)">
+<Target Name="_CreateAndroidDebugSigningKey"
+		Condition="!Exists ('$(_ApkDebugKeyStore)')"
+		DependsOnTargets="$(_OnResolveMonoAndroidSdks)"
```

Comment:
Why is this `DependsOnTargets` required? The only caller -- `_ResolveAndroidSigningKey` -- has `$(_OnResolveMonoAndroidSdks)` within its `DependsOnTargets` anyway.